src/util.cpp

```diff
@@ -697,7 +697,43 @@ boost::filesystem::path GetTempPath() {
 
 void runCommand(std::string strCommand)
 {
-    int nErr = ::system(strCommand.c_str());
+    int nErr = 0;
+#ifdef WIN32
+    // If we're in Windows, use CreateProcess.
+    ULONG nExitCode;
+    STARTUPINFO info={sizeof(info)};
+    ZeroMemory(&info, sizeof(STARTUPINFO));
+    info.cb = sizeof(STARTUPINFO);
+    info.dwFlags = STARTF_USESHOWWINDOW;
+    info.wShowWindow = SW_HIDE;
+    PROCESS_INFORMATION processInfo;
+    ZeroMemory(&processInfo, sizeof(PROCESS_INFORMATION));
+    char system32[256];
+    if (GetSystemDirectory(system32, 255)) {
+        std::stringstream ssCommand;
+        ssCommand << system32 << "\\cmd.exe /C " << strCommand;
+        std::string strTempCmd = ssCommand.str();
+        std::vector<char> vecCommand(strTempCmd.begin(), strTempCmd.end() + 1);
```

Comment:
It is not allowed to use end()+1: end() is the end of the data. There is guarantee that there is anything after it, let alone a \0 byte.

This is becoming quite ugly, but I guess what you could do is (this gets rid of the stringstream too)
```
std::string strTempCmd = std::string(system32) + "\\cmd.exe /C " + strCommand;
const char *sTempCmd = ssCommand.c_str();
std::vector<char> vecCommand(sTempCmd, sTempCmd + strlen(sTempCmd) + 1);
```